jest/private/jest_test.bzl

```diff
@@ -79,3 +96,25 @@ lib = struct(
     attrs = _attrs,
     implementation = _impl,
 )
+
+# TODO: relative_file from bazel-lib breaks because it can't tell the difference
+# between the package name 'jest' in this repo, and the external repository with
+# internal deps names 'jest'. Upstream fix to bazel-lib?
+def _relative_file(ctx, from_file, to_file):
+    workspace_path = to_workspace_path(from_file)
+    path_to_runfiles = "/".join([".."] * len(workspace_path.split("/")))
+    return path_to_runfiles + "/" + to_manifest_path(ctx, to_file)
```

Comment:
@gregmagolan Wdyt of this? I had to update the logic that looks for the sequencer to use this.

Reply:
lgtm